### PR TITLE
fix(ui): GitHubService correct split of remoteUrl in case of alias

### LINKED ISSUE #4132: Failed to load Pull Requests "no repo name specified"
### Version

0.12.5

### Operating System

Mac OS X

### Distribution Method

dmg (Apple Silicon)

### Describe the issue

In a repo, that hosted on a GitHub I cannot create a PR or load a PRs list.

Additional details:
1. remoteUrl is a ssh string with github alias (Because I have multiple GitHub ssh keys, so I've made an alias). Example: `git@arilas.github.com:Arilas/my-repo.git`
2. `GitHubService` line 81: it's incorrectly parse this remoteUrl, so repo is undefined, and owner is `git@arilas`

image

### How to reproduce

1. Create an alias for github in your ssh/config file 
2. Clone a repo using this alias
3. Try to open GitButler
4. Observe errors

### Expected behavior

It should ask user about repo or at least try to parse a string

### Relevant log output

_No response_

### PR DESCRIPTION
**Pull request by Arilas**
## ☕️ Reasoning

If you use a github aliases using SSH to use different SSH keys it's common to just prefix address like `git@arilas.github.com:Arilas/my-repo.git`. But GitButler is incorrectly parse this string, because it's splitting url by `.git`

## 🧢 Changes

Split remoteUrl by last `.git` not the first

## 🎫 Affected issues

Fixes: #4132

With this changes GitHub integration is working for me:

image

### CODE REVIEW

**Review by ndom91 — APPROVED** [review]

**Comment by ndom91** [comment]
Imo we can merge this quick fix for fixing the parsing of a remote URL thats in the ssh alias format, which currently is not parsed correctly.

Since parsing the git URLs is a critical step and there are probably a ton of edge cases, @Byron [recommended](https://github.com/gitbutlerapp/gitbutler/issues/4132#issuecomment-2181391904) leveraging some gitoxide methods that already do this well. I can ask the rust folks here next week about what they think of moving this parsing to rust-land.

Anyway, long story short, thanks for the quick fix - I'll merge this now and it should be available tonight in the `nightly` release :pray:
